requirements-dev.txt

```diff
@@ -6,3 +6,4 @@ isort==5.12.0
 mypy==1.3.0
 pip-tools==7.0.0
 poetry==1.5.1
+pre-commit==3.3.3
```

Comment:
Was this documented somewhere?

Reply:
https://github.com/scaleapi/llm-engine/blob/main/model-engine/README.md?plain=1#L25
we need revamp contributing.md